repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -184,7 +186,19 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
   }
 
   describe("readToDom") {
-    pending
+    it("should return a Document if the stream is able to be parsed") {
+      val document = filter.readToDom(new BufferedServletInputStream(new ByteArrayInputStream(samlResponseStr.getBytes(UTF_8))))
+      document should not be null
+    }
+
+    it("should throw an exception if the stream is not able to be parsed") {
+      val exception = intercept[SamlPolicyException] {
```

Comment:
I do like the way the other one looks though, it's mostly cleaner than the ugly intercept way.